vms/platformvm/api/validator.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package api
+
+import (
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/json"
+	"github.com/ava-labs/avalanchego/vms/platformvm/signer"
+)
+
+// Staker is the representation of a staker sent via APIs.
+// [TxID] is the txID of the transaction that added this staker.
+// [Weight] is the amount of tokens being staked.
+// [StartTime] is the Unix time when they start staking
+// [Endtime] is the Unix time repr. of when they are done staking
+// [NodeID] is the node ID of the staker
+type Staker struct {
+	TxID      ids.ID      `json:"txID"`
```

Comment:
Do we still need these json serialization tags?

Reply:
yeah, this is used in the `GetCurrentValidators` API in platformvm